app/src/main/java/com/rob729/quiethours/Database/Profile.kt

```diff
@@ -14,6 +14,7 @@ data class Profile(
     var smin: Int,
     var ehr: Int,
     var emin: Int,
-    var d: String
+    var d: String,
+    var c: Int
```

Comment:
Please rename the variable to a more comprehendible name, so that other contributors easily get to understand the purpose of this variable.

Reply:
okay, I will do so